### PR TITLE
Robustify dev install instructions in README

### PR DESCRIPTION
**Pull request by fritzo**
- Use http protocol rather than git protocol (which was failing for some internal users)
- Move commit hash down to script so we can copy-and-paste install instructions
- Provide easy suggestion for installing `PyTorch` which often works fine (w/o GPU)

### CODE REVIEW

**Comment by fritzo** [comment]
No need to let tests pass, since this is README only

File: README.md [hunk 1]
```diff
@@ -59,18 +59,22 @@ pip install pyro-ppl[extras]  # for running examples/tutorials
...
 cd pytorch
-git checkout [commit hash above]
+git checkout e40425f  # <---- a well-tested commit
 ```
 Then build PyTorch following instructions in the PyTorch
 [README](https://github.com/pytorch/pytorch/blob/master/README.md).
+Depending on your system, this may be as simple as
+```sh
+pip install ninja
+python setup.py install
```
Comment (jpchen) [inline README.md]: for me the install build fails if i have `ninja`, i had to uninstall `ninja` for build install (vs build develop)

Comment (fritzo) [reply README.md]: @jpchen can you point me to your PR that fixes this by removing this line?

**Comment by jpchen** [comment]
Yep I'll cut a PR I wasn't sure if it was just my machine if you tested it

**Pull request merged**